dev_tools/scripts/generate_cuda_memory_ptx.py

```diff
@@ -28,45 +28,52 @@ class type_desc:
 
 
 memory_spaces = [
-    space(ptx_space_suffix=".shared", ptx_scope_suffix=".cta", fn_suffix="_shared",
-          ptr_expr="convert_generic_ptr_to_smem_ptr({ptr})", ptr_constraint="r"),
-    space(ptx_space_suffix="", ptx_scope_suffix=".gpu", fn_suffix="", ptr_expr="{ptr}", ptr_constraint="l")]
+    space(
+        ptx_space_suffix=".shared",
+        ptx_scope_suffix=".cta",
+        fn_suffix="_shared",
+        ptr_expr="convert_generic_ptr_to_smem_ptr({ptr})",
+        ptr_constraint="r",
+    ),
+    space(
+        ptx_space_suffix="",
+        ptx_scope_suffix=".gpu",
+        fn_suffix="",
+        ptr_expr="{ptr}",
+        ptr_constraint="l",
+    ),
+]
```

Comment:
I prefer the old format in my opinion

Reply:
Right, I think I accidentally formatted with `black` instead of `autopep8`. I'll revert the changes